app/models/unlocalized_words.php

```diff
@@ -0,0 +1,104 @@
+<?php
+namespace Transvision;
+
+// Build arrays for the search form
+$channel_selector = Utils::getHtmlSelectOptions(
+    $repos_nice_names,
+    $repo,
+    true
+);
+$target_locales_list = Utils::getHtmlSelectOptions(
+    Project::getRepositoryLocales($repo),
+    $locale
+);
+
+$ref_locale = Project::getReferenceLocale($repo);
+$unlocalized_words = [];
+
+/*
+    pspell helps getting rid of false positive results by keeping only valid
+    English words. The downside is that it’s filtering out 'jargon' words that
+    can be used in devtools or Mozilla-specific words.
+*/
+if (! $pspell_link = \pspell_new('en_US', '', '', '', PSPELL_FAST)) {
```

Comment:
I would redirect to a 404, but we don't have one yet :-/ In the meantime, we could just add text in the die()?